PlacesOfInterest/app/src/main/java/com/gmail/petraccaro/angelo/placesofinterest/CreateActivity.java

```diff
@@ -142,37 +161,58 @@ public void onClick(View v) {
             }
         });
 
+
+
+
+
         add.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+                String br = bre.getText().toString().trim();
+                String nm = nom.getText().toString().trim();
+                String ds = desc.getText().toString().trim();
+                String item = itemtext.getText().toString().trim();
                 String uriFotoString = null;
-                if( uriFoto != null)
+            if( uriFoto != null || br != null || nm != null || ds != null || item != null) {
```

Comment:
In questa condizione sostituisci con &&. (Considerando che bisogna poi inserire tutti i campi presenti nella condizione per non far fallire l'aggiunta)